src/Payum/Core/Bridge/Propel2/Resources/config/schema.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Based on Propel1 Bridge schema -->
+<database defaultIdMethod="native" name="default">
+    <vendor type="mysql">
+        <parameter name="Engine" value="InnoDB"/>
+        <parameter name="Charset" value="utf8"/>
+    </vendor>
+
+    <table name="payum_order" namespace="Payum\Core\Bridge\Propel2\Model">
```

Comment:
[They also](https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Resources/config/propel/schema.xml#L2) defines a namespace on the database level one time. on the table level they just set a class name and table one